source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,29 @@ The Syscollector module supports different options across various operating syst
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
 |    Solaris             |     ✗     |     √     |     √     |     √    |     ✗     |     ✗     |     –     |
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
+
+.. versionadded:: 4.7.0
+
+The Syscollector module now supports scan packages from `NPM (Node Package Manager)  <https://www.npmjs.com/>`_ and `PyPI (Python Package Index)  <https://pypi.org/>`_, **the support is only for the default paths of those packages' installation**.
+
+The following table shows the new feature that are compatible with various operating systems.
+
++------------------------+----------------------------------------+
+|                        |  **Syscollector package enhancement**  |
++  **Operating System**  +----------------------+-----------------+
+|                        |        NPM           |       PyPI      |
++------------------------+----------------------+-----------------+
+|    Windows             |                  √   |        √        |
++------------------------+----------------------+-----------------+
+|    Linux               |                  √   |        √        |
++------------------------+----------------------+-----------------+
+|    macOS               |                  √   |        √        |
++------------------------+----------------------+-----------------+
+|    FreeBSD             |                  ✗   |        ✗        |
```

Comment:
Are we sure that freebsd and solaris do not have support for those package managers?